concepts/modularization/README.md

```diff
@@ -0,0 +1,83 @@
+# Modularization
+
+Kyma provides Kubernetes building blocks. It should be easy to pick only those that are needed for the job and it should be easy to add new blocks to extend Kyma features. With the growing number of components, it is not possible to always install them all anymore. 
+
+- [Independent releases of modules](#independent-releases-of-modules)
+- [Dependencies between components](#dependencies-between-components)
+- [Release channels](#release-channels)
+- [Versioning](#versioning)
+- [Manifest operator](#manifest-operator)
+
+# Independent releases of modules
+With the growing number of components, it is hard to deliver features and fixes quickly and efficiently. Changes in manifests require a new release of Kyma. Operators (reconcilers) are tightly coupled and must be released together. In most cases, new component releases don't involve any API changes and could be delivered in a few minutes. 
```

Comment:
Does it mean that each team can release a component when it passes unit tests and static checks for example? When do we test kyma integration as a whole?

Reply:
I think this would happen whenever a change is promoted into a channel. THat means as soon as a version gets upgraded in a channel, the tests need to be retriggered for that channel and reject the update in case of failure